test/functional/test_framework/messages.py

```diff
@@ -106,6 +106,10 @@ def uint256_from_str(s):
     return r
 
 
+def uint256_to_string(uint256):
+    return '%064x' % uint256
+
+
```

Comment:
Why should this be in messages.py? why not util.py?

Reply:
I just put it below `uint256_from_string`

Reply:
Oh, that makes sense :D